packages/design-system/src/styles/components/_TooltipIcon.scss

```diff
@@ -42,8 +41,8 @@
 .ds-c-tooltip__trigger:hover,
 .ds-c-tooltip__trigger:focus {
   .ds-c-tooltip-icon__container {
-    border-color: var(--tooltip-icon__color);
```

Comment:
This CSS is sus. The way it's written, the first declaration of `border-color` does nothing so I made an assumption here based on the variable name (`__color`) and changed the CSS rule to follow. I don't know if I'm right, but didn't see any screaming from our VRT.